app/models/miq_server.rb

```diff
@@ -51,19 +51,16 @@ def self.active_miq_servers
   end
 
   def self.atStartup
-    configuration  = VMDB::Config.new("vmdb")
-    starting_roles = configuration.config.fetch_path(:server, :role)
+    starting_roles = ::Settings.server.role
 
     # Change the database role to database_operations
-    roles = configuration.config.fetch_path(:server, :role)
+    roles = starting_roles.dup
     if roles.gsub!(/\bdatabase\b/, 'database_operations')
```

Comment:
In the end this should be a data migration and then we can drop this bit of "live migration" code.  Even so, I won't let this hold up this merge.